quantum/keycode_config.h

```diff
@@ -37,7 +37,7 @@ typedef union {
         bool nkro : 1;
         bool swap_lctl_lgui : 1;
         bool swap_rctl_rgui : 1;
-        bool oneshot_disable : 1;
+        bool oneshot_enable : 1;
```

Comment:
I would guess EEPROM now needs to init to 1 here?

Reply:
yup. commit incoming for that